copy_contacts/copy_contacts.py

```diff
@@ -64,6 +64,10 @@
             log.warning(f"Found a contact in the source instance with multiple URNS. "
                         f"The RapidPro UUID is '{contact.uuid}'")
             continue
+        if contact.urns[0].startswith("tel:") and not contact.urns[0].startswith("tel:+"):
+            log.warning(f"Found a contact in the source instance without a telephone number without a country "
```

Comment:
Do you mean '*with* a telephone number without a country'?

Reply:
Can we count and report the number of cases this applies to? It would be good to know after a copy how many got skipped

Reply:
yes I did and yes we can

Reply:
added